modules/test/playwright/tests/object-web/main/objectEntries.spec.ts

```diff
@@ -2077,59 +2178,160 @@ scheduleTest.describe('Manage object entries schedule properties', () => {
 
 	scheduleTest(
 		'cannot submit an empty displayDate',
-		async ({apiHelpers, page, viewObjectEntriesPage}) => {
-			const objectDefinition =
-				await apiHelpers.objectAdmin.postRandomObjectDefinition({
-					status: {code: 0},
-				});
-
-			apiHelpers.data.push({
-				id: objectDefinition.id,
-				type: 'objectDefinition',
-			});
-
-			await viewObjectEntriesPage.goto(objectDefinition.className);
+		async ({page, viewObjectEntriesPage}) => {
+			await viewObjectEntriesPage.goto(_objectDefinition.className);
 
 			await viewObjectEntriesPage.clickAddObjectEntry(
-				objectDefinition.label['en_US']
+				_objectDefinition.label['en_US']
 			);
 
 			await viewObjectEntriesPage.choosePublicationOption('schedule');
 
+			let requestWasMade = false;
+
+			page.on('request', (request) => {
+				if (request.url().includes(_objectDefinition.restContextPath)) {
+					requestWasMade = true;
+				}
+			});
+
 			await viewObjectEntriesPage.schedulePublicationButton.click();
 
+			// Wait a second before doing the assertion to simulate the time needed for the request to happen
+
+			await page.waitForTimeout(1000);
+
+			expect(requestWasMade).toBe(false);
+
 			await expect(
 				page.getByText('This field is required')
 			).toBeVisible();
+
+			await viewObjectEntriesPage.schedulePublicationCloseButton.click();
 		}
 	);
 
 	scheduleTest(
-		'cannot submit an empty reviewDate when neverReview is not checked',
-		async ({apiHelpers, page, viewObjectEntriesPage}) => {
-			const objectDefinition =
-				await apiHelpers.objectAdmin.postRandomObjectDefinition({
-					status: {code: 0},
+		'cannot submit an empty expirationDate and reviewDate when it is enabled',
+		async ({page, viewObjectEntriesPage}) => {
+			await viewObjectEntriesPage.goto(_objectDefinition.className);
+
+			await viewObjectEntriesPage.clickAddObjectEntry(
+				_objectDefinition.label['en_US']
+			);
+
+			const scheduleLabels = {
+				Expiration: 'Never Expire',
+				Review: 'Never Review',
+			};
+
+			for (const scheduleProperty of ['Expiration', 'Review']) {
```

Comment:
I @larissacribeiro, could you check if we can use ['Expire', 'Review'] and go back to the previous use of template literals.